tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/GroupingTest.java

```diff
@@ -46,7 +45,7 @@ public void testSingleGroupExecution() throws BallerinaTestException {
         String output = balClient.runMainAndReadStdOut("test", args,
                 new HashMap<>(), projectPath, false);
         if (!output.contains(msg)) {
-            AssertionUtils.assertForTestFailures(output, "single group test execution failure");
```

Comment:
Are we still having usages for this method? Can we remove it?

Reply:
We use it when the error output is asserted.